data/dev.js

```diff
@@ -135,3 +135,69 @@ export const people = [
     alt: 'acm logo',
   },
 ];
+
+export const projects = [
+  {
+    name: 'ACM at UCLA Website',
+    prim_lang: 'JS',
+    proj_link: 'https://www.uclaacm.com',
+    repo_link: 'https://github.com/uclaacm/website',
+    description: `The ACM at UCLA Website is the one-stop shop for all things ACM! We develop 
+    and maintain this website, keeping it up-to-date with all the latest news and events. `,
+    img: '/images/dev-projects/acmwebsite.png',
+    alt: 'ACM Website!',
+  },{
+    name: 'Membership Portal',
+    prim_lang: 'JS',
+    proj_link: 'https://members.uclaacm.com/login',
+    repo_link: 'https://github.com/uclaacm/membership-portal',
+    description: `The Membership Portal is where ACM members can check in to events to earn 
+    points! We built the Membership Portal with Node.js, Express.js, PostgreSQL, Redux and React.js.`,
+    img: '/images/dev-projects/membership.png',
+    alt: 'Membership Portal!',
+  },{
+    name: 'Open Source',
+    prim_lang: 'JS',
+    proj_link: 'https://opensource.uclaacm.com/',
+    repo_link: 'https://github.com/uclaacm',
+    description: `Everything we build is open source! In the spirit of open-source, anyone is welcome 
+    to view and contribute to our projects on Github.`,
+    img: '/images/dev-projects/opensource.png',
+    alt: 'Open Source!',
+  },{
+    name: 'CMS Template',
+    prim_lang: 'JS',
+    proj_link: 'https://github.com/uclaacm/committee-website-template-cms',
+    repo_link: 'https://github.com/uclaacm/committee-website-template-cms',
+    description: 'This Committee Website template enables any ACM Committee to put up their website in minutes!',
```

Comment:
can we change description to:
This Committee Website template is a no-code solution that empowers any ACM committee to build and deploy their website in minutes! It's actively in use by multiple committees as it saves time and ensures consistency across sites.